tests/unit/button/button_core_deprecated.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * button_core.js
+ */
+
+
+(function($) {
+
+module( "Button: core deprecated" );
+
+test( "Calling button on a checkbox input calls checkboxradio widget", function(){
+	var checkbox = $( "#checkbox01" );
+
+	expect( 2 );
+	checkbox.button();
+
+	equal( checkbox.is( ":ui-checkboxradio" ), true,
+		"Calling button on a checkbox creats checkboxradio instance" );
+	strictEqual( checkbox.checkboxradio( "option", "icon" ), false,
```

Comment:
Same here, though mixing `equal` and `strictEqual` next to each other is not a great approach anyway.